ts/components/services/ServiceMetadata.tsx

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import { View, StyleSheet } from "react-native";
+
+import I18n from "../../i18n";
+import { TranslationKeys } from "../../../locales/locales";
+import { H4 } from "../core/typography/H4";
+import TouchableDefaultOpacity from "../TouchableDefaultOpacity";
+import { capitalize } from "../../utils/strings";
+import customVariables from "../../theme/variables";
+import { ItemAction } from "../../utils/url";
+
+import ItemSeparatorComponent from "../ItemSeparatorComponent";
+
+import SectionHeader from "./SectionHeader";
+import LinkRow from "./LinkRow";
+
+type Props = {
+  address?: string;
+  androidStoreUrl?: string;
+  email?: string;
+  fiscalCode?: string;
+  getItemOnPress: (value: string, valueType?: ItemAction) => () => void;
+  iosStoreUrl?: string;
+  pec?: string;
+  phone?: string;
+  serviceId: string;
+  supportUrl?: string;
+  webUrl?: string;
+};
+
+const styles = StyleSheet.create({
+  valueHeader: {
+    marginTop: customVariables.spacerLargeHeight
+  },
+  valueRow: {
+    flexDirection: "column"
+  },
+  valueItem: {
+    flexDirection: "row",
+    flex: 1,
+    justifyContent: "space-between",
+    marginVertical: 24
+  },
+  itemLabel: {
+    flex: 1
+  },
+  itemValue: {
+    flex: 1,
+    marginRight: 8
+  }
+});
+
+/**
+ * Renders a dedicated section with a service's metadata and the header.
+ */
+const ServiceMetadata: React.FC<Props> = ({
+  address,
+  androidStoreUrl,
+  email,
+  fiscalCode,
+  getItemOnPress,
+  iosStoreUrl,
+  pec,
+  phone,
+  serviceId,
+  supportUrl,
+  webUrl
+}: Props) => (
+  <>
+    <SectionHeader iconName="io-phone" title={"services.contactsAndInfo"} />
+
+    {/* links */}
+    {webUrl && <LinkRow text={"services.visitWebsite"} href={webUrl} />}
+    {supportUrl && (
+      <LinkRow text={"services.askForAssistance"} href={supportUrl} />
+    )}
+    {iosStoreUrl && (
+      <LinkRow text={"services.otherAppIos"} href={iosStoreUrl} />
+    )}
+    {androidStoreUrl && (
+      <LinkRow text={"services.otherAppAndroid"} href={androidStoreUrl} />
+    )}
+
+    {/* touchable rows */}
+    {fiscalCode &&
+      renderInformationRow(
+        fiscalCode,
+        "profile.fiscalCode.fiscalCode",
```

Comment:
Should it be "C.F. Ente"? 
![Schermata 2021-07-14 alle 17 54 24](https://user-images.githubusercontent.com/822471/125653115-d21138f2-b2b3-4b95-a32a-9e69089220a2.png)

Now it is "Codice Fiscale" (it is coming from profile)

Reply:
There's a "Codice Fiscale Ente Creditore" in the Payment Notice screen—we can 'stick' to that string and change it into "Codice Fiscale Ente" ;)

Reply:
Correct, fixed https://github.com/pagopa/io-app/pull/3225/commits/674959d7e7f5aff731686fd54c2c6be99c6280e1